### ISSUE TITLE
New stretchoid IPs collected January 2021 to July 2022.

### ISSUE THREAD

**Issue by GWHAYWOOD** [SilvrrGIT/IP-Lists#7]
A few more IPs for your stretchoid list:

192.241.192.20  192.241.192.232 192.241.198.173 192.241.202.59  192.241.203.171 192.241.204.28  192.241.205.143 192.241.205.218 192.241.205.220 192.241.205.75
192.241.206.137 192.241.206.31  192.241.206.84  192.241.206.94  192.241.207.116 192.241.207.129 192.241.207.8   192.241.208.16  192.241.209.77  192.241.210.174
192.241.211.225 192.241.212.111 192.241.212.126 192.241.212.128 192.241.212.202 192.241.212.249 192.241.212.251 192.241.213.101 192.241.213.120 192.241.213.131
192.241.213.153 192.241.213.213 192.241.213.224 192.241.213.236 192.241.213.237 192.241.213.56  192.241.213.90  192.241.214.107 192.241.214.142 192.241.214.17
192.241.214.18  192.241.214.250 192.241.214.251 192.241.214.47  192.241.214.63  192.241.214.92  192.241.215.104 192.241.215.27  192.241.215.42  192.241.216.130
192.241.216.165 192.241.216.78  192.241.217.131 192.241.217.221 192.241.217.43  192.241.218.13  192.241.218.20  192.241.218.229 192.241.218.240 192.241.218.96
192.241.219.110 192.241.219.123 192.241.219.128 192.241.219.157 192.241.219.159 192.241.219.161 192.241.219.227 192.241.219.44  192.241.219.55  192.241.219.56
192.241.219.62  192.241.219.83  192.241.220.119 192.241.220.153 192.241.220.186 192.241.220.220 192.241.220.229 192.241.220.26  192.241.220.32  192.241.220.94
192.241.221.111 192.241.221.132 192.241.221.138 192.241.221.154 192.241.221.196 192.241.221.204 192.241.221.240 192.241.221.38  192.241.221.51  192.241.221.8
192.241.221.96  192.241.222.11  192.241.222.114 192.241.222.133 192.241.222.136 192.241.222.163 192.241.222.182 192.241.222.20  192.241.222.213 192.241.222.25
192.241.222.4   192.241.222.48  192.241.222.5   192.241.223.189 192.241.223.243 192.241.224.12  192.241.224.151 192.241.224.213 192.241.224.61  192.241.224.89
192.241.225.140 192.241.225.146 192.241.225.20  192.241.225.56  192.241.225.88  192.241.226.103 192.241.226.127 192.241.226.8   192.241.226.84  198.199.96.229

Sorry, I haven't yet managed to get my browser to attach a file (nor most other things) on Github.

**Comment by SilvrrGIT** [comment]
The list came back as non Stretchoid IPs, primarily generic Digital Ocean servers.  The rest will be added, thanks for the contribution!
192.241.198.173
192.241.205.75
192.241.208.16
192.241.210.174
192.241.211.225
192.241.214.107
192.241.214.63
192.241.214.92
192.241.215.104
192.241.216.165
192.241.217.221
192.241.217.43
192.241.218.13
192.241.218.20
192.241.218.229
192.241.218.96
192.241.223.189
192.241.223.243
192.241.224.12
192.241.224.151
192.241.224.213
192.241.224.61
192.241.224.89
192.241.225.140
192.241.225.146
192.241.225.20
192.241.225.56
192.241.225.88
192.241.226.103
192.241.226.127
192.241.226.8
192.241.226.84

**Comment by GWHAYWOOD** [comment]
Curious, they were definitely stretchoid when they connected to our mail servers.  Perhaps there's some churn.


Would you like a real-time feed?

**Comment by SilvrrGIT** [comment]
There is some churn, I did a re-verification of my list and had to remove a significant number of IPs that no longer were associated with the stretchoid.com domain.  

I used (https://www.infobyip.com/ipbulklookup.php) to look the addresses up in bulk (up to 100 at a time) and found they returned the same results as other sites I have used in the past before adding/removing an IP.  I went through 500+ IP addresses this AM so Im sure I could have made an error so let me know if you find any. 

I primarily use these lists to remove some noise from my firewall logs so I don't think I need a real-time feed.  I do appreciate any contributions you have as IPs pop up in your logs though.
